builds/e2e/templates/e2e-run.yaml

```diff
@@ -10,7 +10,7 @@ steps:
     # Filter out flaky tests.
     $filter = 'Category!=Flaky'
 
-    if ('$(arch)' -eq 'arm32v7' -Or '$(arch)' -eq 'arm64v8')
+    if ('$(arch)' -eq 'arm32v7' -Or '$(arch)' -eq 'aarch64')
```

Comment:
In master and release/1.2, it looks like the established pattern is to use 'arm64v8'. The terms 'arm64v8' and 'aarch64' are both correct and more or less interchangeable (as far as I know), so you should probably stick to the current convention of using 'arm64v8' in this YAML script.

Reply:
Done, Updated the E2E Test Pipeline link in the Description as wel